src/SFA.DAS.FAT.Domain/Courses/QarModel.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+
+namespace SFA.DAS.FAT.Domain.Courses;
+
+public sealed class QarModel
+{
+    public string Period { get; set; }
+    public string Leavers { get; set; }
+    public string AchievementRate { get; set; }
+    public string NationalLeavers { get; set; }
+    public string NationalAchievementRate { get; set; }
+    public int TotalParticipants => TotalParticipantCount();
+    public decimal FailureRate => GetFailureRate();
+    public string QarPeriodStartYear { get => $"20{Period.AsSpan(0, 2)}"; }
+    public string QarPeriodEndYear { get => $"20{Period.AsSpan(2, 2)}"; }
+    public string PeriodDisplay => $"{QarPeriodStartYear} to {QarPeriodEndYear}";
+
+    public int ConvertedLeavers
+    {
+        get
+        {
+            if (int.TryParse(Leavers, out int value))
+            {
+                return value;
+            }
+
+            return 0;
+        }
+    }
+
+    public decimal? ConvertedAchievementRate
+    {
+        get
+        {
+            if (decimal.TryParse(AchievementRate, out decimal value))
+            {
+                return value;
+            }
+
+            return null;
+        }
+    }
+
+    private int TotalParticipantCount()
```

Comment:
If TotalParticipantCount is the total completers of a course, then the value is just ConvertedLeavers

We might have to discuss this